api/graphite.go

```diff
@@ -38,6 +38,10 @@ var MissingOrgHeaderErr = errors.New("orgId not set in headers")
 var MissingQueryErr = errors.New("missing query param")
 var InvalidFormatErr = errors.New("invalid format specified")
 var InvalidTimeRangeErr = errors.New("invalid time range requested")
+var TooManySeriesErr = response.NewError(
+	http.StatusRequestEntityTooLarge,
```

Comment:
i was going to say, this is only for request entities (requests from client, e.g. file uploads), but we do the same for `errMaxPointsPerReq`, so ok...